hwy/ops/set_macros-inl.h

```diff
@@ -221,7 +237,49 @@
 
 #define HWY_NAMESPACE N_PPC8
 
-#define HWY_TARGET_STR "altivec,vsx"
+#define HWY_TARGET_STR HWY_TARGET_STR_PPC8
+
+//-----------------------------------------------------------------------------
+// PPC9
+#elif HWY_TARGET == HWY_PPC9
+
+#define HWY_ALIGN alignas(16)
+#define HWY_MAX_BYTES 16
+#define HWY_LANES(T) (16 / sizeof(T))
+
+#define HWY_HAVE_SCALABLE 0
+#define HWY_HAVE_INTEGER64 1
+#define HWY_HAVE_FLOAT16 0
+#define HWY_HAVE_FLOAT64 1
+#define HWY_MEM_OPS_MIGHT_FAULT 1
+#define HWY_NATIVE_FMA 1
+#define HWY_CAP_GE256 0
+#define HWY_CAP_GE512 0
+
+#define HWY_NAMESPACE N_PPC9
+
+#define HWY_TARGET_STR HWY_TARGET_STR_PPC9
+
+//-----------------------------------------------------------------------------
+// PPC10
```

Comment:
It seems fine to have a separate ----- section for PPC8..10, or we could consider putting them in one section and adding `#if` only where they differ, i.e. HWY_NAMESPACE/TARGET_STR? (Similar example is HWY_SVE2/HWY_SVE)

Reply:
Can combine the HWY_PPC8/HWY_PPC9/HWY_PPC10 into a single section and adding ```#if``` only where they differ